content/applications/productivity/voip/axivox/call_queues.rst

```diff
@@ -2,172 +2,117 @@
+A call queue organizes and routes incoming calls when all agents are busy, placing callers on hold
+in the order they called. This system helps manage high call volumes more efficiently, ensures fair
+workload distribution, and provides a more predictable experience for both callers and agents.
 
-The callers then wait on hold to be connected to the next available call center agent.
-
-Implementing a call queue system reduces stress for employees, and helps build brand trust with
-customers. Many companies use call queues to set expectations with customers, and to distribute the
-workload equally amongst employees.
-
-This document covers the process required to configure call queues (with advanced settings), as well
-as how to log into a call queue from the Odoo database.
+This document explains how to configure call queue settings and log into a queue from the Odoo
+database.
 
 .. seealso::
-   :ref:`voip/axivox/music_on_hold`
+   :ref:`Set up on hold music <voip/axivox/music_on_hold>`
 
 Add a queue
 ===========
 
 To add a call queue in Axivox, navigate to the `Axivox management console
 <https://manage.axivox.com>`_. In the left menu, click :guilabel:`Queues`. Next, click
-:guilabel:`Add a queue`. Doing so reveals a blank :guilabel:`New queue` form with various fields to
-fill out.
-
-Name
-----
-
-Once the :guilabel:`New queue` page appears, enter the :guilabel:`Name` of the queue.
-
-Internal extension
-------------------
-
-Choose an :guilabel:`Internal extension` for the queue. This is a number to be dialed by users of
-the database to reach the login prompt for the queue.
-
-Strategy
---------
-
-Next, is the :guilabel:`Strategy` field. This field determines the call routing of received calls
-into this queue.
-
-The following choices are available in the :guilabel:`Strategy` drop-down menu:
-
-- :guilabel:`Call all available agents`
-- :guilabel:`Calls the agent who has received the call for the longest time`
-- :guilabel:`Calls the agent who has received the least call`
-- :guilabel:`Call a random agent`
-- :guilabel:`Call agents one after the other`
-- :guilabel:`Call agents one after the other starting with the first in the list`
-
-Choose a strategy that best meets the company's needs for customers in the queue.
-
-Maximum waiting time in seconds
--------------------------------
-
-In the :guilabel:`Maximum waiting time in seconds` field, determine the longest time a customer
-waits in the queue before going to a voicemail, or wherever else they are directed to in a dial
-plan. Enter a time in seconds.
-
-Maximum duration of ringing at an agent
----------------------------------------
-
-In the :guilabel:`Maximum duration of ringing at an agent` field, determine the longest time an
-individual agent's line rings before moving on to another agent, or moving to the next step in the
-dial plan. Enter a time in seconds.
+:guilabel:`Add a queue`. From here, set up the call queue.
+
+- **Name**: The call queue's name. A required field.
```

Comment:
Im curious why these are not using :guilabel: formatting, since they are files on the current UI, just like the drop-down options listed below here 🤔

Please make sure use `:guilabel:` markup to identify any text of the interactive user interface: 
https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#gui-element
```suggestion
- :guilabel:`Name`: The call queue's name. A required field.
```